pxteditor/editorcontroller.ts

```diff
@@ -253,6 +257,10 @@ export function bindEditorMessages(getEditorAsync: () => Promise<IProjectView>)
                                 return Promise.resolve()
                                     .then(() => projectView.toggleGreenScreen());
                             }
+                            case "toggleaccessibleblocks": {
```

Comment:
Should this be `togglekeyboardcontrols` and should similar naming be updated everywhere?